airbyte-cdk/python/airbyte_cdk/sources/declarative/incremental/resumable_full_refresh_cursor.py

```diff
@@ -94,3 +95,15 @@ def get_request_body_json(
         next_page_token: Optional[Mapping[str, Any]] = None,
     ) -> Mapping[str, Any]:
         return {}
+
+
+@dataclass
+class ChildPartitionResumableFullRefreshCursor(ResumableFullRefreshCursor):
```

Comment:
as per @girarda 's suggestion, let's add a docstring here explaining what we talked about below about the purpose of the class and why this cursor exists